operations/observability/mixins/cross-teams/rules/components/nodes/alerts.libsonnet

```diff
@@ -29,6 +29,38 @@
                 ) - 10 < 5
               |||,
           },
+          {
+            alert: 'GitpodNodeConntrackTableIsFull',
+            labels: {
+              severity: 'critical',
+            },
+            'for': '5m',
+            annotations: {
+              runbook_url: 'https://github.com/gitpod-io/runbooks/blob/main/runbooks/GitpodNodeConntrackTableIsFull.md',
```

Comment:
Rock on, @sagor999 !

May I ask you to [add a related runbook for this critical alert](https://github.com/gitpod-io/runbooks)? 

It's Gitpod policy to have a runbook for critical alerts, because they page. Warnings do not page (through PagerDuty) and do not require a runbook.

Reply:
Forgot to link it here, here is runbook for it: https://github.com/gitpod-io/runbooks/pull/320